Example/SmileID/HomeViewModel.swift

```diff
@@ -86,6 +86,13 @@ class HomeViewModel: ObservableObject, SmartSelfieResultDelegate, DocumentCaptur
                                                object: nil)
     }
 
+    func didSucceed(selfie: Data,
+                     documentFrontImage: Data,
+                    documentBackImage: Data?,
+                    jobStatusResponse: JobStatusResponse?) {
```

Comment:
Why is this Optional?

Reply:
In a situation where the selfie capture flow is used just for capturing. There will be no job status response. I realised this while using the SelfieCaptureView for DocV.

Reply:
I don't think I follow. There should still be a response here from the docv job